Submodels/output_handling.py

```
@@ -0,0 +1,18 @@
+"""
+
+
+"""
+
+import os
+
+class output_handling:
+    def __init__(self, *args):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)